Allure.SpecFlowPlugin/PluginConfiguration.cs

```diff
@@ -1,79 +1,133 @@
+    //    public Regex TestMethodRegex { get; }
 
-            ParentSuiteRegex = ParseRegex(configuration["specflow:grouping:suites:parentSuite"]);
-            SuiteRegex = ParseRegex(configuration["specflow:grouping:suites:suite"]);
-            SubSuiteRegex = ParseRegex(configuration["specflow:grouping:suites:subSuite"]);
+    //    public Regex OwnerRegex { get; }
+    //    public Regex SeverityRegex { get; }
 
-            EpicRegex = ParseRegex(configuration["specflow:grouping:behaviors:epic"]);
-            StoryRegex = ParseRegex(configuration["specflow:grouping:behaviors:story"]);
+    //    public Regex IssueRegex { get; }
+    //    public Regex TmsRegex { get; }
 
-            PackageRegex = ParseRegex(configuration["specflow:grouping:packages:package"]);
-            TestClassRegex = ParseRegex(configuration["specflow:grouping:packages:testClass"]);
-            TestMethodRegex = ParseRegex(configuration["specflow:grouping:packages:testMethod"]);
 
-            OwnerRegex = ParseRegex(configuration["specflow:labels:owner"]);
-            SeverityRegex = ParseRegex(configuration["specflow:labels:severity"]);
+    //    public PluginConfiguration(string allureConfiguration)
+    //    {
+    //        var sf = JObject.Parse(allureConfiguration)["specflow"];
 
-            IssueRegex = ParseRegex(configuration["specflow:links:issue"]);
-            TmsRegex = ParseRegex(configuration["specflow:links:tms"]);
+    //        var stepArguments = sf["stepArguments"];
+    //        ConvertToParameters = stepArguments["convertToParameters"].Value<bool>();
+    //        ParamNameRegex = ParseRegex((string)stepArguments["paramNameRegex"]);
+    //        ParamValueRegex = ParseRegex((string)stepArguments["paramValueRegex"]);
 
+    //        var grouping = sf["grouping"];
+    //        ParentSuiteRegex = ParseRegex((string)grouping["suites"]["parentSuite"]);
+    //        SuiteRegex = ParseRegex((string)grouping["suites"]["suite"]);
+    //        SubSuiteRegex = ParseRegex((string)grouping["suites"]["subSuite"]);
 
-        }
+    //        EpicRegex = ParseRegex((string)grouping["behaviors"]["epic"]);
+    //        StoryRegex = ParseRegex((string)grouping["behaviors"]["story"]);
 
-        private Regex ParseRegex(string value)
-        {
-            if (string.IsNullOrWhiteSpace(value))
-                return null;
-            try
-            {
-                return new Regex(value,
-                    RegexOptions.Compiled | RegexOptions.Singleline | RegexOptions.IgnoreCase);
+    //        PackageRegex = ParseRegex((string)grouping["packages"]["package"]);
+    //        TestClassRegex = ParseRegex((string)grouping["packages"]["testClass"]);
+    //        TestMethodRegex = ParseRegex((string)grouping["packages"]["testMethod"]);
 
-            }
-            catch (Exception)
-            {
-                return null;
-            }
-        }
-    }
+    //        var labels = sf["labels"];
```

Comment:
whats the point of keeping the code commented?

Reply:
Massive code review drawback ;( Fixed. Thanks!